packages/stockflux-components/src/index.css

```diff
@@ -22,13 +22,45 @@
 
   --chart-teal: #37d8bc;
   --chart-teal-lighter: #51f2d6;
```

Comment:
Does these need to move to go with the other `chart` items?

Reply:
Those are the older colours, which will be removed once the charts are updated and then reference the new colours only.
I kept them separate from the new section to make sure we can tell which is which when removing them later, or if the value gets changed to the new one we can move them down into the `UX Colours` section.

Reply:
Apologies, you did mention that. Maybe the older ones could be grouped under an outmoded styles comment and then a separate ticket raised for removing them?

Reply:
Moved old colours to end of file, with comment above that they are not to be used. When full UX work is completed they can be removed.